### PR TITLE
Migrate to Core v0.11

### PR DESCRIPTION
**Pull request by LVala**
I have one doubt, when changing `{{:error, reason}, state}` to `raise ...` I used `raise "Error: #{inspect(reason)}"` everywhere and I'm wondering whether that's enough or should I make the error messages more verbose.

### CODE REVIEW

File: lib/membrane_h264_ffmpeg/parser.ex [hunk 1]
```diff
@@ -266,38 +264,41 @@ defmodule Membrane.H264.FFmpeg.Parser do
...
     )
   end
 
   @impl true
-  def handle_caps(:input, %Membrane.H264.RemoteStream{}, ctx, _state)
+  def handle_stream_format(:input, %Membrane.H264.RemoteStream{}, ctx, _state)
       when ctx.pads.input.start_of_stream?,
-      do: raise("Cannot send Membrane.H264.RemoteStream caps after the stream has started")
+      do:
+        raise("Cannot send Membrane.H264.RemoteStream stream_format after the stream has started")
```
Comment (bblaszkow06) [inline lib/membrane_h264_ffmpeg/parser.ex:275]: In addition to reformatting, I'd rephrase the error a bit:
```suggestion
    raise "Cannot handle Membrane.H264.RemoteStream format after the stream has started"
  end
```

**Review by bblaszkow06 — CHANGES_REQUESTED** [review]

**Comment by varsill** [comment]
@LVala @bblaszkow06  Would it be possible to update the dependency to membrane_h264_format to 0.4.0?

File: mix.exs [hunk 2]
```diff
@@ -40,16 +40,16 @@ defmodule Membrane.H264.FFmpeg.Plugin.MixProject do
     [
       {:bunch, "~> 1.3"},
       {:unifex, "~> 1.0"},
-      {:membrane_core, "~> 0.10.0"},
-      {:membrane_common_c, "~> 0.13.0"},
+      {:membrane_core, "~> 0.11.0"},
+      {:membrane_common_c, "~> 0.14.0"},
       {:membrane_h264_format, "~> 0.3.0"},
```
Comment (bblaszkow06) [inline mix.exs:45]: As requested, pls update the format package
```suggestion
      {:membrane_h264_format, "~> 0.4.0"},

```

**Comment by LVala** [comment]
Because of the new line `{:ok, :unknown} -> raise "Unknown H264 profile!"` in `get_profile!` function, some tests do not pass, that will be investigated.

**Comment by FelonEkonom** [comment]
@LVala this new line also causes errors in RTP Plugin tests

**Review by bblaszkow06 — APPROVED** [review]
